### PR TITLE
Add Void Linux build instructions

### PR DESCRIPTION
**Pull request by maciejla**
Signed-off-by: Maciej <git@maciej.website>

Added dependencies and notes for building on Void Linux

<details>
<summary>Checklist</summary>

- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [ ] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

</details>

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -184,6 +184,31 @@ of Tenacity's dependencies:
...
+
+To install Tenacity's dependencies, run:
+```
+sudo xbps-install cmake ninja lame-devel libsndfile-devel libsoxr-devel portaudio-devel gtk+3-devel wxWidgets-gtk3-devel portmidi-devel libid3tag-devel libmad-devel twolame-devel soundtouch-devel ffmpeg-devel lv2 lilv-devel
+```
+
+##### Notes
+
+- Void Linux does not package PortSMF and SBSMS. This means MIDI and SBSMS timestretching will not be available.
+If you need these features, it's recommended to build them from source.
```
Comment (nbsp) [inline BUILDING.md:197]: Maybe link to the source or to some documentation?

File: BUILDING.md [hunk 2]
```diff
@@ -184,6 +184,31 @@ of Tenacity's dependencies:
...
+
+- wxWidgets 3.1 is suggested but not packaged in Void Linux. Refer to the
+[wxWidgets documentation](https://docs.wxwidgets.org/3.1/overview_cmake.html)
+for how to install it from source code, or see the [previous section](#wxwidgets-from-source).
+
+- Alternatively, you can skip installing wxWidgets 3.1 and use 3.0 instead:
+```
+sudo xbps-install wxWidgets-gtk3-devel
+```
+If you do so, you will need to set `WX_CONFIG` environment variable:
```
Comment (nbsp) [inline BUILDING.md]: nitpick

```suggestion
If you do so, you will need to set the `WX_CONFIG` environment variable:
```

File: BUILDING.md [hunk 3]
```diff
@@ -184,6 +184,31 @@ of Tenacity's dependencies:
...
+```
+
+##### Notes
+
+- Void Linux does not package PortSMF and SBSMS. This means MIDI and SBSMS timestretching will not be available.
+If you need these features, it's recommended to build them from source.
+
+- wxWidgets 3.1 is suggested but not packaged in Void Linux. Refer to the
+[wxWidgets documentation](https://docs.wxwidgets.org/3.1/overview_cmake.html)
+for how to install it from source code, or see the [previous section](#wxwidgets-from-source).
```
Comment (nbsp) [inline BUILDING.md:201]: Would it make sense to list wx3.1's build dependencies here?

**Review by nbsp — CHANGES_REQUESTED** [review]

File: BUILDING.md (hunk 1 above)
Comment (maciejla) [reply BUILDING.md:197]: I don't know what I should link to, I just wasn't able to find these packages in void linux repo

File: BUILDING.md (hunk 3 above)
Comment (maciejla) [reply BUILDING.md:201]: Probably yes, I'll do that

**Comment by n0toose** [comment]
Hi, just circling back to this, thank you so much for working on this once again. Do you plan to finish this anytime soon and if not, should another person try and take over from here (we're all fairly busy, so it'll take a while either way)?

**Comment by maciejla** [comment]
> Hi, just circling back to this, thank you so much for working on this once again. Do you plan to finish this anytime soon and if not, should another person try and take over from here (we're all fairly busy, so it'll take a while either way)?

Hi, unfortunately I don't have much time right now :( I'll try to finish this in a reasonable time, but if someone wants to take over, feel free to do it :)

**Pull request closed without merging**